src/device-registry/bin/kafka-connect.js

```diff
@@ -0,0 +1,8 @@
+const dotenv = require("dotenv");
+dotenv.config();
```

Comment:
In case this is supposed to be a bash script, please consider removing the file extension and adding a[ shebang](https://en.wikipedia.org/wiki/Shebang_(Unix)) at the top of the file accordingly

Reply:
@Baalmart this has been fixed